dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -334,6 +334,19 @@ ClassPathEntry findClassLocation(String className) {
     return Iterables.getFirst(classFileNameToClassPathEntry.get(specialLocation), null);
   }
 
+  /**
+   * Returns class file name which is a location of a class file within a JAR file. Usually it is
```

Comment:
"Returns class file name" --> Converts a binary name to a class file name.

It is **not** a location. It is a name.

Reply:
Updated.